jdk/make/lib/SoundLibraries.gmk

```diff
@@ -152,9 +152,8 @@ else
   endif
 
   ifeq ($(OPENJDK_TARGET_CPU), aarch64)
-    LIBJSOUND_CFLAGS += -DX_ARCH=X_AARCH64
+	LIBJSOUND_CFLAGS += -DX_ARCH=X_AARCH64
```

Comment:
We don't really want this change: the leading tab could be problematic.

Reply:
sure, we can keep initial JCL patch instead.